constructor/header.sh

```diff
@@ -531,6 +531,7 @@ if [ "${PYTHONPATH:-}" != "" ]; then
 fi
 
 if [ "$BATCH" = "0" ]; then
+    DEFAULT=no
```

Comment:
I guess this makes writing these "if" statements easier, but I think that the "generated code" looks uglier.

I do personally prefer the addition of the full if statement:

```
#if initialize_conda is False
   DEFAULT=no
#endif
````

Reply:
The DEFAULT variable is not used in the case of `initialize_conda is False`, see if statement in line 541/542... so also "ugly" :/ ... ` ... anyways no change at all required for the header.sh due to `initialize_by_default` now being initialised to False instead of None. 

A general pattern has to be enabled in the code so that the values which are read via `info.get("name", "default")` actually have a default and type defined in the construct.py.

Reply:
understood. then lets not modify the header.sh at all :D.